wallet-connect/src/client.rs

```diff
@@ -235,10 +256,17 @@ pub enum WCError<M: Middleware> {
     ClientError(ClientError),
 }
 
-impl<M: Middleware> FromErr<M::Error> for WCError<M> {
-    fn from(src: M::Error) -> WCError<M> {
+impl<M: Middleware> MiddlewareError for WCError<M> {
+    type Inner = M::Error;
```

Comment:
from_err, as_inner is pair, maybe add comment?

Reply:
those are functions from ethers upgraded trait - MiddlewareError, document would be in the trait.